app/views/ordertimes/_form.html.haml

```diff
@@ -44,7 +44,7 @@
 
   = f.labeled_boolean_field(:billable, detail: 'Leistung ist verrechenbar.')
   - if Settings.meal_compensation.present?
-    = f.labeled_boolean_field(:meal_compensation, detail: "Ich war vor Ort beim Kunden und habe Anrecht auf die #{link_to('Verpflegungsentschädigung', Settings.meal_compensation.url)}.".html_safe)
+    = f.labeled_boolean_field(:meal_compensation, detail: "Ich war beim Kunde vor Ort und möchte die #{link_to('Verpflegungs-Entschädigung', Settings.meal_compensation.url)} erhalten.".html_safe)
```

Comment:
1. beim Kunde**n**
2. Warum der Bindestrich beim zusammengesetzten Substantiv? Verpflegungsentschädigung ist richtiger (ausser, wenn hier die Schweizer Schriftsprache stark vom Deutschen Schriftsprache abweicht).

Reply:
Isch scho gmörtscht dänk

Reply:
Verpflegungsentschädigung in einem Wort finde ich okay, dann aber bitte überall anpassen:

https://github.com/puzzle/puzzletime/search?q=Verpflegungs-Entsch%C3%A4digung&unscoped_q=Verpflegungs-Entsch%C3%A4digung